### DISCUSSION TITLE
New Features

### DISCUSSION THREAD

**Discussion by isaiahbjork** [isaiahbjork/Auto-GPT-MetaTrader#6] [Ideas]
Share your ideas on new features! 📈

**Comment by Rikiar73574** [comment]
i suggest replacing that list of commands
``rsi
money_flow_index
volume
sma
ema
macd
wma
adx
adi
fib_retracements
stochastic_oscillator
tsi
Fundamentals``

with a single command to get all possible data in a single stroke, it often seems that gpt gets confused and forgets those.

**Reply by Nairb99** [reply]
how do you do this
